cmake/std/atdm/cts1/environment.sh

```diff
@@ -0,0 +1,35 @@
+################################################################################
+#
+# Set up env on cts1 for ATMD builds of Trilinos
+#
+# This source script gets the settings from the ATDM_CONFIG_BUILD_NAME var.
+#
+################################################################################
+
+# Chama and cts1 jobs all use the same environmnet changes to the
+# sourced script below will impact jobs on both of those
+# machines. please be mindful of this when making changes
+
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" == "DEFAULT" ] ; then
+  export ATDM_CONFIG_KOKKOS_ARCH=BDW
+fi
+
+if [ "$ATDM_CONFIG_KOKKOS_ARCH" != "BDW" ] ; then
+  echo "***"
+  echo "*** ERROR: KOKKOS_ARCH=$ATDM_CONFIG_KOKKOS_ARCH is not a valid option on this system."
+  echo "*** '$ATDM_CONFIG_KOKKOS_ARCH' appears in $ATDM_CONFIG_BUILD_NAME which then sets the KOKKOS_ARCH"
+  echo "*** on cts1 'BDW' is the only valid KOKKOS_ARCH. If no KOKKOS_ARCH is specified then"
+  echo "*** 'BDW' will be used by default"
+  echo "***"
+  return
+fi
+
+export ATDM_CONFIG_SPARC_TPL_BASE=/projects/sparc/tpls/cts1-bdw
+
+export ATDM_CONFIG_SBATCH_DEFUALT_TIMEOUT=4:00:00
```

Comment:
At least on eclipse, this will restrict jobs to one partition that accepts this time limit.

Reply:
> ATDM_CONFIG_SBATCH_DEFUALT_TIMEOUT

Looks like I mispelled "DEFAULT".  I will push a commit to fix this.